binfmt/binfmt_loadmodule.c

```diff
@@ -194,7 +194,7 @@ int load_module(FAR struct binary_s *bin, FAR const char *filename,
 
                   /* Free the allocated fullpath */
 
-                  kmm_free(fullpath);
+                  lib_free(fullpath);
```

Comment:
but binfmt is called only from kernel mode, the usage should be unified to kmm_malloc/kmm_free.

Reply:
but the implementation of envpath is in libs/libc/misc/lib_envpath.c, lib_malloc/lib_free will be more reasonable.